modules/core/src/smithy4s/http/HttpUnaryClientCodecs.scala

```diff
@@ -118,6 +119,8 @@ object HttpUnaryClientCodecs {
     override def withRawHttpLabelValues(enabled: Boolean): Builder[F, Request, Response] =
       copy(rawHttpLabelValues = enabled)
 
+    def isRawHttpLabelValues: Boolean = rawHttpLabelValues
```

Comment:
nitpick: `has` instead of `is`. Frankly I'd rename the whole concept to `urlEncodingHttpLabels` or something like that, but it's too late for now / should be done on 0.19